app/src/main/java/jp/developer/retia/frozenword/ui/habbits/HabbitsViewModel.kt

```diff
@@ -22,12 +22,20 @@ class HabbitsViewModel @Inject constructor(
     val events: SharedFlow<HabbitsEvent> = _events.asSharedFlow()
 
     init {
+        loadAndEmitState()
+    }
+
+    private fun loadAndEmitState() {
         viewModelScope.launch(ioDispatcher) {
             val habbits = habbitRepository.getHabbitAndLogs()
             _uiState.emit(HabbitsUiState.Loaded(habbits))
         }
     }
 
+    fun onResume(){
```

Comment:
```suggestion
    fun onResume() {
```



<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>